lib/recurly/paypal/strategy/complete.js

```diff
@@ -0,0 +1,32 @@
+import { PayPalStrategy } from './index';
+
+// const debug = require('debug')('recurly:paypal:strategy:complete');
```

Comment:
Are we actually using this for debugging or can we remove it?

Reply:
we removed!

Reply:
Good catch. Thanks.